src/cmd/create.rs

```diff
@@ -62,9 +61,9 @@ pub struct Sharded {
     /// Number of shards required to recover the key
     recovery_threshold: u8,
 
-    #[structopt(long, possible_values = &["bip39", "mobile"], case_insensitive = true, conflicts_with = "swarm")]
+    #[structopt(long, conflicts_with = "swarm")]
     /// Use a BIP39 or mobile app seed phrase to generate the wallet keys
-    seed: Option<SeedType>,
+    seed: bool,
```

Comment:
since this is no longer an `Option`, plain `seed` for the name is less clear. Someone like `with_seed` would be more clear

Reply:
IIRC, `seed` what the original command line option from before I added any of this seed type stuff last year and I aimed for a small a change as possible to make it easier to merge and rebase.

I can change it, but IMO it's more clear as `seed`, especially that we also now have `swarm` in the code instead of `with_swarm`. I guess both could change?

Reply:
I prefer `with_`, but you have a point about `swarm`, so I'll defer this to another PR